src/cli/Cli.ts

```diff
@@ -661,57 +664,78 @@ export class Cli {
     return undefined;
   }
 
-  private printHelp(helpMode: string, exitCode: number = 0): void {
+  private async handleHelp(helpMode: string, exitCode: number = 0): Promise<void> {
     const properties: any = {};
 
     if (this.commandToExecute) {
       properties.command = this.commandToExecute.name;
-      this.printCommandHelp(helpMode);
+      const pathChunks = this.retrievePathChunks();
+      const helpTarget = this.getSettingWithDefaultValue<string>(settingsNames.helpTarget, Cli.defaultHelpTarget);
+
+      if (helpTarget === Cli.defaultHelpTarget) {
```

Comment:
In the next line, we assume a specific condition: showing help in the console. But here, we're comparing against the default config value. Should we change the default in the future, then this comparison will lead to an incorrect behavior. To avoid it, let's compare `helpTarget` against a specific value.